Cargo.toml

```diff
@@ -10,64 +10,3 @@ members = [
 	"runtime/*",
 	#"pallets/*",
 ]
-
-# The list of dependencies below (which can be both direct and indirect dependencies) are crates
-# that are suspected to be CPU-intensive, and that are unlikely to require debugging (as some of
-# their debug info might be missing) or to require to be frequently recompiled. We compile these
-# dependencies with `opt-level=3` even in "dev" mode in order to make "dev" mode more usable.
-# The majority of these crates are cryptographic libraries.
-#
-# Note that this does **not** affect crates that depend on Substrate. In other words, if you add
-# a dependency on Substrate, you have to copy-paste this list in your own `Cargo.toml` (assuming
-# that you want the same list). This list is only relevant when running `cargo build` from within
-# the Substrate workspace.
-#
-# If you see an error mentioning "profile package spec ... did not match any packages", it
-# probably concerns this list.
-#
-# This list is ordered alphabetically.
-[profile.dev.package]
```

Comment:
Remove the compile optimization option in dev mode, it will make the grcov command fail.
The error is:
```
Unsupported instrumentation profile format version
```

Refer to the substrate-parachain-template project, which does not have these configuration items in its Cargo.toml.
https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/Cargo.toml